gcs/CONFIGURATION.md

```diff
@@ -309,7 +309,7 @@ default service account impersonation.
 
     Enables Cloud Storage direct uploads.
 
-*   `fs.gs.outputstream.sync.min.interval.ms` (default: `0`)
+*   `fs.gs.outputstream.sync.min.interval.ms` (default: `10,000`)
```

Comment:
Reverted this change - will create a separate PR for defaults change. We can do this because we plan a new major 3.0.0 GCS connector release that can have breaking changes.